content/hardware/07.opta/opta-family/opta/tutorials/01.user-manual/content.md

```diff
@@ -1211,6 +1211,184 @@ You should be able now to connect to your Opta™ using a central device. The Bl
 
 ![Bluetooth® Low Energy service and characteristic information from an Opta™ device](assets/user-manual-14.png)
 
+## OPC Unified Architecture (OPC UA)
+
+This section explains using the [**OPC UA library**](https://github.com/arduino-libraries/Arduino_OPC_UA) designed for Opta.

```

Comment:
@TaddyHC Am I the only one who finds this sentence weird? Looks like it is "correct" but would like to make we do not have problems

```suggestion
This section explains the use of the [**OPC UA library**](https://github.com/arduino-libraries/Arduino_OPC_UA) designed for Opta.

```